_blinkr/jenkins_blinkr_runner.rb

```diff
@@ -0,0 +1,94 @@
+require_relative '../_docker/lib/process_runner'
+require 'fileutils'
+require 'json'
+
+#
+# This class wraps a number of calls to _docker/control.rb to allow us to run the broken link checks and amalgamate the result into a generalised fail or pass
```

Comment:
Can you update the code comments here so that it accurately reflects what the code does to avoid confusion.